### ISSUE TITLE
2.0.2 (ERROR) Unexpected character encountered while parsing value: P. Path '', line 0, position 0.

### ISSUE THREAD

**Issue by vfuyong** [PocketMobsters/MobBotTemp#101]
[21:21:23] (ERROR) Unexpected character encountered while parsing value: P. Path '', line 0, position 0.
{"Command":"GetPokemonSettings"}
EggsList
[21:21:27] (ERROR) Unexpected character encountered while parsing value: E. Path '', line 0, position 0.
[21:21:27] (CATCH) Throwing Spinning Nice! PokeBall
{"Command":"GetTrainerProfile"}
InventoryList
[21:21:34] (ERROR) Error parsing Infinity value. Path '', line 1, position 2.

**Comment by m-unkel** [comment]
Do you use NecrobotVisualizer ? There is a problem related to NecrobotVisualizer, but it is not a bug of MobBot.

**Comment by PureGore** [comment]
You can use PokeEase which is a visualizer made to work with pokemobbot.

**Comment by vfuyong** [comment]
Yeah I confirmed this is triggered by necrobotvisualizer. Thanks for the tips.

**Comment by vfuyong** [comment]
@PureGore Can I have the link to PokeEase?

**Comment by PureGore** [comment]
https://github.com/GediminasMasaitis/PokeEase
